app/addons/permissions/components.react.jsx

```diff
@@ -123,39 +123,73 @@ function (app, FauxtonAPI, React, Components, Stores, Actions) {
       this.setState({newRole: e.target.value});
     },
 
+    getAccountBlurb: function () {
+      var blurb = '';
+      if (this.props.section === 'admins') {
+        blurb = 'Grant database ADMIN privileges to the following user accounts:';
+      } else {
+        blurb = 'Grant database member privileges to user accounts with the following roles:';
+      }
+      return (
+        <header>
+          <h4 className="capitalize">{this.props.section} Accounts</h4>
+          <p>{blurb}</p>
+        </header>
+      );
+    },
+
+    getRoleBlurb: function () {
+      var blurb = '';
+      if (this.props.section === 'admins') {
+        blurb = 'Grant database ADMIN privileges to user accounts with the following roles:';
+      } else {
+        blurb = 'Grant database member privileges to user accounts with the following roles:';
+      }
+      return (
+        <header>
+          <h4 className="capitalize">{this.props.section} Roles</h4>
+          <p>{blurb}</p>
+        </header>
+      );
+    },
+
+    getPageHeader: function () {
+      if (this.props.section === 'admins') {
+        return (
+          <header className="page-header">
+            <h3>{this.props.title}</h3>
+            <p className="help">
+              {this.getHelp()}
+              <a className="help-link" data-bypass="true" href={getDocUrl('DB_PERMISSION')} target="_blank">
+                <i className="icon-question-sign"></i>
+              </a>
+            </p>
+          </header>
+       );
+      } else {
+        return '';
```

Comment:
you can use an early return instead. 

please also return `null` instead of an empty string